docs/pages/core/wireguard-mesh.md

```diff
@@ -0,0 +1,116 @@
+
+If the direct link above does not work, go to [https://material-digital.de/contact/](https://material-digital.de/contact/), log in and choose the subject "Network and security architecture".
+
+Your request MUST include at least the following information:
+  - The public (sub)domain which points to your PMD-S' public IPv4 address
+  - Your country code (DE, FR, UK, US etc.)
+  - Name of your company/institute (Fraunhofer ABC, KIT, BAM etc.)
+  - Name of your business unit (IT, Materials Department etc.)
+
+If everything is alright, you will receive a ZIP file containing your identity certificate and key file.
+
+### Adjust Firewall
+
+* Make sure your public IP can be reached via **port 29292** for **both UDP and TCP**.
+
+### Check IPv6 is Enabled on Your System
+
+* Check with e.g. `sysctl net.ipv6.conf.all.disable_ipv6`, which should show 0.
+  Otherwise enable it (typically with `sysctl net.ipv6.conf.all.disable_ipv6=0` and by making changes persistent through editing `/etc/sysctl.conf`).
+
+### Check Wireguard (Kernels < 5.6 only!)
+
+* When using a **kernel version < 5.6**, check that the wireguard kernel module is available and loaded:
+
+```
+sudo modprobe wireguard 
+sudo lsmod | grep wireguard
+```
+
+### Get the Repository
+
+If not done yet, download/clone this (`pmd-server`) repository on your PMD-S.
+
+```
+git clone https://github.com/materialdigital/pmd-server.git
+cd pmd-server
+```
+
+## Setup
+
+### Automatic Setup using PMD Setup Container
+- Place the ZIP file containing your identity certificate/key files in the mesh directory (i.e. `wg-mesh`).
+- Change (`cd`) into the mesh directory.
+- *When migrating from an old setup, do `docker compose down` to stop the currently running mesh containers and remove the old Docker network via `docker network rm wgnet`.*
+- Run `docker run --rm --pull always -v $(pwd):/composer_root -v /run/docker.sock:/run/docker.sock materialdigital/setup` from within this directory.
+- Check the output for setup warnings and resolve them accordingly.
+
+### Manual Setup (NOT Recommended!)
+- Place your identity certificate/key files under `pmd_config/wg/` and rename them to `participant.{cert/key}`.
+- Create the wgnet Docker network with `docker network create --ipv6 --subnet=172.31.0.0/16 --subnet=<your-wg-mesh-subnet> wgnet`, using the IPv6 subnet which has been assigned to you.
+- Create/edit the `.env` file, and adapt `WG_ENDPOINT`, `SUBNET_PREFIX` and `PMD_ZONE` to the values which have been assigned to you. You should normally not alter the `PMDC_SUBNET_PREFIX` variable.
+- *In case subnet `172.31.0.0/16` has not been avialable, you also have to specify `IPV4_DNS_IP` (IPv4 IP of DNS service) and `IPV4_SUBNET` (IPv4 subnet of `wgnet` in CIDR notation).*
```

Comment:
```suggestion
- *In case subnet `172.31.0.0/16` has not been available, you also have to specify `IPV4_DNS_IP` (IPv4 IP of DNS service) and `IPV4_SUBNET` (IPv4 subnet of `wgnet` in CIDR notation).*
```